src/reducer.js

```diff
@@ -153,7 +154,8 @@ const RegistrationLiteReducer = (state = DEFAULT_STATE, action) => {
         }
         case UPDATE_CLOCK: {
             const { timestamp } = payload;
-            return { ...state, nowUtc: timestamp };
+            const allowedTicketTypes = state.requestedTicketTypes ? state.ticketTypes.filter((tt) => (tt.sales_start_date === null && tt.sales_end_date === null) || (timestamp >= tt.sales_start_date && timestamp <= tt.sales_end_date)) : null;
```

Comment:
remove this
the issue seems to be the intial value of nowUtc
its on millis and should be on second
please do use
moment().unix()  as initial value 
https://www.geeksforgeeks.org/moment-js-moment-unix-function/